server/database/config.js

```diff
@@ -0,0 +1,117 @@
+
+import { Pool } from 'pg';
+import bcrypt from 'bcrypt';
+import dotenv from 'dotenv';
+import { DB_URL } from '../../db';
+
+const debug = require('debug')('config');
+
+dotenv.config();
+
+// debug('NODE_ENV', process.env.NODE_ENV);
+console.log('NODE_ENV', process.env.NODE_ENV);
+
+/**
+ * class DB
+ */
+class DB {
+  /**
+   * construtor
+   */
+  constructor() {
+    this.connection = new Pool({
+      connectionString: DB_URL,
+    });
+  }
+
+  /**
+   * createTables
+   */
+  async createTables() {
+    console.log('CONNECTING TO DATABASE: DATABASE URL', DB_URL);
+    this.connection.on('connect', () => {
```

Comment:
@musonant What happens if the connection fails?

Reply:
I will add the listener for an error 
```
this.connection.on('error', (err, client) => {
    console.error('Unexpected error', err);
    process.exit(-1);
});
```
Will that do? @tosmak16